Dockerfile

```diff
@@ -20,4 +20,4 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 ENV PATH="/app/.venv/bin:$PATH"
 EXPOSE 8080
 
-CMD ["gunicorn", "--worker-class", "uvicorn.workers.UvicornWorker", "wsgi:app", "-b", "0.0.0.0:8080"]
+CMD ["gunicorn", "--worker-class", "eventlet", "wsgi:app", "-b", "0.0.0.0:8080"]
```

Comment:
As discussed https://eventlet.readthedocs.io/en/latest/#warning

Reply:
Thanks for catching this Steven, really good find 🙏

We need to switch to _something_ other than Uvicorn, because that is designed for ASGI apps - which Connexion was. Native Flask is a WSGI app (and `gunicorn` is a WSGI server), so we have a few options for workers:

1) `sync` - fully synchronous, 1 request per worker at a time
3) `eventlet` - have generally seen this used across other government apps (eg Notify), but as you say the docs suggest not using it for new things. Had a bunch of issues getting monkeypatching happening early enough, and have been scarred a few times using eventlet in the past so I think we're good to avoid this.
2) `gevent` - also green thread based, should be a better option than `sync` and does seem to just be working without having to fiddle too hard.

So will go for `gevent`.